pokemon/pokemon.py

```diff
@@ -0,0 +1,79 @@
+"""This module contains a two classes for creating pokemon."""
+
+
+class Pokemon(object):
+    """
+    This class Pokemon represents a pokemon character with several attributes.
+
+     :param name: Name of pokemon
+     :type name: str
+     :param nickname: Nickname of pokemon
+     :type nickname: str
+     :param moves: list of 4 moves
+     :type moves: list
+    """
+
+    def __init__(self, name, nickname):
+        """Initialise Pokemon class."""
+        self.name = name
+        self.nickname = nickname
+        self.moves = ["jump", "strike", "dash", "block"]
```

Comment:
Couple of points:

1. This docstring format is valid, and perfectly fine if you're working on your own. We have an informal convention though, to use numpy-style docstrings, all of our other modules do it. See here for the format: https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_numpy.html
2. I prefer not to tell the type of the parameters in the docstring. Recent versions of Python support type hints, as I've indicated in the suggestion below. I find that this is better practice for two reasons:
  a. `mypy` can work with it and help you out as you code, and 
  b. when you're maintaining the code later, and you need to modify something, it's much easier to update the type hint and keep it correct when it's right there in your code, rather than in the docstring.
3.  "Parameters" strictly means the things between the brackets `(name, nickname)`. You have another one, `moves`, which gets initialised but it isn't one of the parameters that you pass to the `__init__` method. You can still document this, but you would put it under the heading of `Attributes`. I know how to do that with Numpy style, I'm not really sure about the convention that you've used here.

```suggestion
    """
    This class Pokemon represents a pokemon character with several attributes.

     :param name: Name of pokemon
     :param nickname: Nickname of pokemon
    """

    def __init__(self, name: str, nickname: str):
        """Initialise Pokemon class."""
        self.name = name
        self.nickname = nickname
        self.moves = ["jump", "strike", "dash", "block"]
```